_inc/client/components/footer/index.jsx

```diff
@@ -170,8 +178,7 @@ export class Footer extends React.Component {
 					<li className="jp-footer__link-item">
 						<a
 							onClick={ this.trackPrivacyClick }
-							href="https://automattic.com/privacy/"
-							target="_blank"
+							href="#/privacy"
```

Comment:
I noticed this link doesn't work from this list of all modules page:
`/wp-admin/admin.php?page=jetpack_modules`

![2018-03-16_08-59-54](https://user-images.githubusercontent.com/1563559/37534558-06ccea08-28fa-11e8-8876-041839d23305.png)

Reply:
Good catch @jaswrks!

Since this is the JS-version of Jetpack settings page, I think we'll want to add a new JS-free admin page that contains the texts and links plus a checkbox and submit button to disable tracking.

Thoughts @dereksmart?

Reply:
Yeah, let's take that on in another PR